docs/src/mlj_cheatsheet.md

```diff
@@ -89,57 +93,75 @@ Same as above but exclude `:Time` column from `X`:
 using RDatasets
 channing = dataset("boot", "channing")
 y, X =  unpack(channing,
-               ==(:Exit),            # y is the :Exit column
-               !=(:Time);            # X is the rest, except :Time
+               ==(:Exit),     # y is the :Exit column
```

Comment:
The comment makes it an overly long line for the cheat sheet. Should either exclucde the comment or write it after the code block.
```suggestion
               ==(:Exit),     
```